### PR TITLE
Modified the type of NetworkId to uint32_t

### PR DESCRIPTION
**Pull request by BonnieLo**
Modified the type of NetworkId to uint32 to follow the definition of MCTP/Endpoint.interface in phosphor-dbus-interfaces.

Test log:
- Check the type of NetworkId is 'u'.

root@bmc:~# busctl introspect xyz.openbmc_project.MCTP \
            /xyz/openbmc_project/mctp/1/40
NAME                                TYPE      SIGNATURE RESULT/VALUE FLAGS
au.com.CodeConstruct.MCTP.Endpoint  interface -         -            -
.Remove                             method    -         -            -
.SetMTU                             method    u         -            -
org.freedesktop.DBus.Introspectable interface -         -            -
.Introspect                         method    -         s            -
org.freedesktop.DBus.Peer           interface -         -            -
.GetMachineId                       method    -         s            -
.Ping                               method    -         -            -
org.freedesktop.DBus.Properties     interface -         -            -
.Get                                method    ss        v            -
.GetAll                             method    s         a{sv}        -
.Set                                method    ssv       -            -
.PropertiesChanged                  signal    sa{sv}as  -            -
xyz.openbmc_project.MCTP.Endpoint   interface -         -            -
.EID                                property  y         40           const
.NetworkId                          property  u         1            const
.SupportedMessageTypes              property  ay        2 0 1        const

### CODE REVIEW

**Comment by jk-ozlabs** [comment]
Looks good, thanks. A couple of changes I would request though, just for the commit log:

* Could you add your signed-off-by line?
* Include a prefix of which areas the change modifies (`mctpd`, in this case)
* (and this is super minor) could you rephrase the log message to be a description of what the change *does*: ie, in the present tense. I would suggest something like:

```
mctpd: Change the type of NetworkId to uint32_t

Change NetworkId to be a uint32 (dbus type 'u'), to follow the definition of MCTP/Endpoint.interface
in phosphor-dbus-interfaces
```

but details are up to you, of course :)

**Comment by DelphineCCChiu** [comment]
Hi @jk-ozlabs ,
Modified the commit log with your suggestions.

**Comment by RickyWu-wiwynn** [comment]
Hi @jk-ozlabs ,
Modified the commit log with your suggestions. Thanks.

**Comment by jk-ozlabs** [comment]
Thanks, merged!
